util/b58-decoder/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "mc-util-b58-decoder"
 version = "4.0.2"
 authors = ["MobileCoin"]
 edition = "2021"
+license = "GPL-3.0"
```

Comment:
This is probably client code

Reply:
I was a bit surprised, but this isn't used in either of the SDKs anywhere.

Reply:
I feel like it might be used in full-service.
I see, you are mainly concerned about the SDKs as opposed to all clients

Reply:
Yeah, `mobilecoind` is still GPL-3.0, e.g.